torch_xla/test/metrics_test_wrapper.py

```diff
@@ -112,21 +152,72 @@ def _write_to_disk(output_string, output_filename):
   if not FLAGS.root or not FLAGS.test_folder_name:
```

Comment:
If these are required, I'd use required arguments.
Example:

https://github.com/pytorch/xla/blob/4a33f7ce62f3da58690e83c4e3ad97871d00e1f7/scripts/gen.py#L1036

Reply:
Added `required=True` to both args and removed the ValueError